custom/icds_reports/utils/aggregation_helpers/distributed/child_health_monthly.py

```diff
@@ -92,13 +92,16 @@ def _state_aggregation_query(self, state_id):
         alive_in_month = "(child_health.date_death IS NULL OR child_health.date_death - {} >= 0)".format(
             start_month_string
         )
-        migration_status = "(agg_migration.is_migrated = 1 AND agg_migration.migration_date::date < {" \
-                           "start_month_string})::integer".format(start_month_string=start_month_string)
-        registered_status = "(agg_availing.is_registered = 0 AND agg_availing.registration_date::date < {" \
-                            "start_month_string})::integer".format(start_month_string=start_month_string)
-        seeking_services = "({registered_status} IS DISTINCT FROM 1 AND " \
-                           "{migration_status} IS DISTINCT FROM 1)".format(registered_status=registered_status,
-                                                                           migration_status=migration_status)
+        migration_status = (
```

Comment:
It might be clearer also to name these "migrated" and "registered" (or i guess in this case not_registered) so that it is obvious what the boolean means. "migration_status" does not tell you whether True means migrated or not migrated